src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/IncrementalBuildTest.cs

```diff
@@ -12,6 +13,29 @@ namespace Xamarin.Android.Build.Tests
 	[Parallelizable (ParallelScope.Children)]
 	public class IncrementalBuildTest : BaseTest
 	{
+		[Test]
+		public void CheckNothingIsDeletedByIncrementalClean ([Values (true, false)] bool enableMultiDex)
+		{
+			// do a release build
+			// change one of the properties (say AotAssemblies) 
+			// do another build. it should NOT hose the resource directory.
+			var path = Path.Combine ( "temp", TestName );
+			var proj = new XamarinAndroidApplicationProject () {
+				ProjectName = "App1",
+				IsRelease = true,
+			};
```

Comment:
Should we make this a forms project like this test: https://github.com/xamarin/xamarin-android/blob/e6044b0c1f664243d496390b9f7c3263bd4176f8/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs#L311-L328

Maybe should also add a custom view?

And that opens up another question, should we create a new `XamarinFormsApplicationProject`class? I think this code is starting to get duplicated in various tests.